.travis.yml

```diff
@@ -82,8 +82,6 @@ before_install:
   - composer install
   # NPM Version
   - npm i -g npm@5.7
-  # CoffeeScript
-  - npm install coffee-formatter
```

Comment:
I don't see where this is being used. Maybe old. If it passes tests then I suppose it is OK to remove 👍 .

Reply:
The tests pass, and I don't see it being used anywhere in the build process either.
It's not in `appveyor.yml`; but I find it weird that it passes in OSX and not in Linux